.eslintrc.cjs

```diff
@@ -16,9 +16,10 @@ module.exports = {
   parserOptions: {
     project: ['./tsconfig.*json'],
   },
-  plugins: ['react-refresh', 'cypress'],
+  plugins: ['react-refresh', 'cypress', 'eslint-plugin-tsdoc'],
   rules: {
     'react-refresh/only-export-components': ['warn', { allowConstantExport: true }],
     'react/react-in-jsx-scope': 'off',
+    'tsdoc/syntax': 'warn',
```

Comment:
was this on "error" before? If so, should we not better leave it on error?

Reply:
No, I took this directly from their `usage` docs on their [npm page](https://www.npmjs.com/package/eslint-plugin-tsdoc?activeTab=readme).
It makes sense to only warn about TSDoc since it's not going to affect the app if it doesn't adhere to the conventions.